doc/data_structures.rst

```diff
@@ -2,22 +2,24 @@
 Data Structures
 ***************
 
-RedisAI defines three new data structures to be
-used in redis databases: tensor, model, and script.
-In addition, SmartRedis defines an additional data
-structure ``DataSet``.  In this section, the SmartRedis
-API for interacting with these data structures
-will be described, and when applicable,
-comments on performance and best practices will be made.
-
-In general, concepts and capabilities will be
-demonstrated for the Python and C++ API.
-The C and Fortran function signatures closely
-resemble the C++ API, and as a result,
-they are not discussed in detail in the interest
-of brevity.  For more detailed explanations of the C
-and Fortran API, refer to the documentation pages for those
-clients.
+RedisAI defines three data structures designed for use within Redis databases: 
+
+* ``Tensor`` : represents an n-dimensional array of values.
+* ``Model`` : represents a computation graph by one of the supported DL/ML framework backends.
```

Comment:
replace "computation graph.." with "computational ML model for one of the supported backend frameworks"